src/serac/physics/integrators/displacement_hyperelastic_integrator.cpp

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/physics/integrators/displacement_hyperelastic_integrator.hpp"
+
+#include "serac/infrastructure/profiling.hpp"
+#include "serac/numerics/expr_template_ops.hpp"
+#include "serac/numerics/array_4D.hpp"
+#include "serac/physics/utilities/physics_utils.hpp"
+
+namespace serac::mfem_ext {
+
+void DisplacementHyperelasticIntegrator::CalcKinematics(const mfem::FiniteElement&    element,
```

Comment:
All these comments may be useless as they will be OBE when Sam's changes go in.

But probably the better way to do the classes here is:

displacementsolidintegrator:hyperelastickinematics

Then you can mix material models with different kinematics within the same problem

Reply:
I called it `HyperelasticIntegrator` as it can only use hyperelastic materials at the moment. I think your naming scheme is better though.